Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/stock_attachments.yml

```diff
@@ -74,6 +74,48 @@
       recoilFlat: 2
       scatterFlat: 6
 
+- type: entity
+  parent: RMCStockAttachmentBase
+  id: RMCAttachmentXM51Stock
+  name: XM51 stock
+  description: A specialized stock designed for XM51 breaching shotguns. Helps the user absorb the recoil of the weapon while also reducing scatter. Integrated mechanisms inside the stock allow use of a devastating two-shot burst. This comes at a cost of the gun becoming too unwieldy to holster, worse handling and mobility.
+  components:
+  - type: Sprite
+    state: xm51_stock
+  - type: AttachableVisuals
+    suffix: ""
+  - type: Tag
+    tags:
+    - RMCAttachmentStock
+    - RMCAttachmentXM51Stock
+  - type: AttachableSpeedMods
+    modifiers:
+    - conditions:
+        wieldedOnly: true
+      walk: -0.059
+      sprint: -0.101
+  - type: AttachableWieldDelayMods
+    modifiers:
+    - delay: 0.4
+  - type: AttachableWeaponMeleeMods
+    modifiers:
+    - bonusDamage:
+        types:
+          Blunt: 10
+  - type: AttachableWeaponRangedMods
+    modifiers:
+    - conditions:
+        wieldedOnly: true
+      accuracyAddMult: 0.2
+      recoilFlat: -2
+      scatterFlat: -6
+    - conditions:
+        unwieldedOnly: true
+      accuracyMovementPenaltyAddMult: -1
+      accuracyAddMult: -0.15
+      recoilFlat: 2
+      scatterFlat: 6
+
```

Comment:
Doublecheck this, please. Both of the accuracy modifiers are wrong, as is the unwielded scatter. Burst fire addition is missing. See the BFA in `under_attachments.yml` for how to add it.

Reply:
Kinda tweaked the numbers, but I can't add the BFA for the shotgun as it requires pump before firing it